utils/set_dockwidget_gui.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this whole logic should be moved to ``dockwidget.py``, and into the dockwidget class itself. Basically, you're operating on the dockwidget, and if that's the case, than we probably should group that logic together.

Additional points:
- you're repeating the same things three times, better to use a loop
- I think a more elegant solution is to do something like this (more reusable also):

```python
def set_status_bar_text(status_bar, text):
    status_bar.setText(text)
    status_bar.repaint()
```

- ``status_bar_text`` as a function name is unclear to me what it does. Only after reading the code would you understand that it sets all three status bars.